Sources/PayPalDataCollector/Public/PayPalDataCollector/PPDataCollector.swift

```diff
@@ -41,10 +67,22 @@ import Security
         return result?.getPayPalClientMetaDataId() ?? ""
     }
 
+    // NEXT_MAJOR_VERSION: make this not a class function
+    @objc class func generateClientMetadataID(isSandbox: Bool) -> String {
+        PPDataCollector.generateClientMetadataID("", disableBeacon: false, isSandbox: isSandbox, data: nil)
+    }
```

Comment:
Duplication of this function below because `@selector` can't pass in parameters. We should be able to clean this up in next major version so we don't need 3 functions doing basically the same thing.